docs/examples/index.md

```diff
@@ -1384,3 +1421,123 @@ To parse the logs, one easy way is to use [jq](https://stedolan.github.io/jq/) -
+
+The list of drugs and medications to annotate is based on a publicly available list of FDA-approved drugs and active ingredients. The data can be downloaded directly from [Drugs@FDA database](https://www.fda.gov/drugs/informationondrugs/ucm079750.htm). The list of used drugs is stored as `drug.lst` and active ingredients as `active.lst`.
+
+More information about creating custom GATE applications can be found in the official [GATE documentation](https://gate.ac.uk/documentation.html).
+
+### Data preparation
+
+This example is a basic one as our NLP application is not fully context aware and is looking in the text only for the words found in the provided list of drugs. Moreover, as the list of drugs and active ingredients can contain full names of drugs (or ingredients), the full names of drugs may not be found in the parsed text. Hence, the FDA-approved list of drugs and components needs to be further post-processed and filtered. All this is being performed by script `extra/clean_list.py` in the example directory. It uses as an input the raw data downloaded from Drugs@FDA database and a list of words to filter out (e.g., the most frequent 10000 words in English) to prepare a curated list of drugs and active ingredients.
+
+[//]: # "<span style='color:red'> NOTE: </span>"
+**Note: For a more reliable list of drugs one should better refer to [UMLS RxNorm](https://www.nlm.nih.gov/research/umls/rxnorm/) resource, however a special UMLS license applies for the usage and/or redistribution of that resource.**
+
+
+## Properties file
+
+The *properties* file used in this example is based on the one from [Example 2](#example-2), but extended with parts using GATE documents processor and which are covered below.
+
+
+### Spring profiles
+
+The spring profile part has been updated with adding a `gate` profile:
+```properties
+spring.profiles.active=jdbc_in,elasticsearchRest,gate,localPartitioning
+```
+
+### GATE configuration
+
+A new part covering documents processing using a custom GATE application has been added:
+```properties
+gate.gateHome = /gate/home/
+gate.gateApp = /gate/app/drug.gapp
+gate.fieldsToGate = encounter_document
+gate.gateAnnotationTypes = Drug
+gate.gateFieldName = gate
+```
+The property `gate.gateHome` denotes the home directory of GATE application, which should be the same for all GATE applications when using CogStack GATE image from Dockerhub (please see below). `gate.gateApp` denotes the name of the GATE application to be run -- in our example, the application directory (containing the *gapp* application with resources) will be directly mounted into CogStack container into `/gate/app/` directory.
```

Comment:
gapp or xgapp ?

Reply:
Seems that both conventions are fine according to [GATE Documentation](https://gate.ac.uk/sale/tao/splitch3.html):
    A common convention is to use .gapp or .xgapp as a ﬁle extension.

I will reformulated a bit the sentences.